web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/IiifV3ManifestControllerTest.java

```diff
@@ -141,4 +144,72 @@ public void testGetCanvas() throws Exception {
         var items = (List) respMap.get("items");
         assertFalse(items.isEmpty());
     }
+
+    @Test
+    public void testGetManifestWithAVFiles() throws Exception {
+        var workObj = new ContentObjectSolrRecord();
+        workObj.setId(OBJECT_ID);
+        workObj.setResourceType(ResourceType.Work.name());
+        workObj.setTitle("Test Work");
+        when(accessCopiesService.listViewableFiles(eq(OBJECT_PID), any())).thenReturn(Arrays.asList(workObj));
+
+        var result = mockMvc.perform(get("/iiif/v3/" + OBJECT_ID + "/manifest")
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andReturn();
+
+        Map<String, Object> respMap = MvcTestHelpers.getMapFromResponse(result);
+        assertEquals("Manifest", respMap.get("type"));
+        assertEquals("http://example.com/iiif/v3/f277bb38-272c-471c-a28a-9887a1328a1f/manifest", respMap.get("id"));
+        assertEquals("Test Work", ((List) ((Map) respMap.get("label")).get("none")).get(0));
```

Comment:
Just a note that you could use getResponseAsJson here and maybe save some casting, but that's totally up to you.

Reply:
Ah, I think this was copy pasted from the test that was already there. I'll take a look and clean up, I think it's not supposed to be here?